docs/Concepts.md

```diff
@@ -126,6 +126,11 @@ pull in your environment, so we provide a way for you to configure that by edit
 the `image`'s value in a configmap named
 [`config-entrypoint`](./../config/config-entrypoint.yaml).
 
+### Cluster Task
+
+Similar to `Task` but with a cluster-wide scope. Cluster Tasks are available in 
```

Comment:
Can we provide some examples of Cluster tasks to users? Like what kind of tasks could be cluster task Vs a normal Task.
Why shdn't all tasks be cluster tasks?

Reply:
@tejal29 any task can be a ClusterTask, it depends on how operators set up their clusters, if they want their users to just use standard tasks available for everybody in all namespaces, or want users to setup their own. It is similar to ClusterBuildTemplates and BuildTemplates in Build

Reply:
one thing that i noticed in the most recent user study is that i think we've started mixing some of our using docs into the concepts docs - I'd like to move some of the stuff we've got in `concepts.md` out and keep this doc pretty short and sweet, just about the high level concepts

something we can iterate on tho! and i think it might make sense to write up some guidelines on which docs go where 🤔